tests/integration/targets/ec2_transit_gateway/tasks/main.yml

```diff
@@ -123,6 +123,7 @@
         filters:
           options.dns-support: enable
           options.vpn-ecmp-support: enable
+          options.multicast-support: enable
```

Comment:
I'm suggesting to remove `options.multicast-support: enable` from here and add:
- task to create a new ec2 transit gateway with multicast support (enabled or disabled)
- use the ec2_transit_gateway_info module with this filter options.multicast-support: enable or disable
- ensure the transit gateway you created is deleted inside the always block

Reply:
Done, I inserted these two parts:

    - name: test create transit gateway with multicast enabled
      ec2_transit_gateway:
        description: "{{ tgw_description }}"
        multicast_support: true
      register: create_result
    - name: assert changed is True
      assert:
        that:
          - create_result.changed == True
          
    - name: test success with multicast enabled
      ec2_transit_gateway_info:
        filters:
          options.multicast-support: enable
      register: result
    - name: assert success with transit_gateway_id filter
      assert:
        that:
          - 'result.changed == false'
          - 'result.transit_gateways != []